client/app/dedicated/server/firewall/dedicated-server-firewall.routes.js

```diff
@@ -1,13 +1,15 @@
 import template from './dedicated-server-firewall.html';
 
-angular.module('App').config(/* @ngInject */($stateProvider) => {
-  $stateProvider.state('app.dedicated.server.firewall', {
-    url: '/firewall',
-    views: {
-      'tabView@app.dedicated.server': {
-        template,
+angular.module('App').config(/* @ngInject */($stateProvider, coreConfigProvider) => {
+  if (coreConfigProvider.region !== 'US') {
+    $stateProvider.state('app.dedicated.server.firewall', {
+      url: '/firewall',
+      views: {
+        'tabView@app.dedicated.server': {
+          template,
+        },
       },
-    },
-    translations: { value: ['..'], format: 'json' },
-  });
+      translations: { value: ['..'], format: 'json' },
+    });
+  }
```

Comment:
Don't know if it's better to have a `redirectTo` to handle this that will redirect to `app.dedicated.server` for US instead